docs/pages/deploy-a-cluster/teleport-enterprise/hsm.mdx

```diff
@@ -57,7 +57,7 @@ On the EC2 instance where you will run your Teleport Auth Server:
 
 3. Start the CloudHSM client.
    ```code
-   $ sudo service start cloudhsm-client
+   $ sudo service cloudhsm-client start
```

Comment:
This change is accurate, but for consistency, should we update it to use the `systemctl` format?

```suggestion
   $ sudo systemctl start cloudhsm-client
```